common/cmake/gen.cmake

```diff
@@ -182,8 +182,7 @@ function(N_TEMPLATE)
   #   NAME <name>
   #   SRCS <list of sources>
   #   PREFIXES <list of prefixes>
-  #   [APPLY_V2X]
-  #   [APPLY_VERILOG_IMAGE_GEN]
+  #   [GENERATED | SCANNER_TYPE <verilog|xml>]
```

Comment:
This is a mistake.  The way it was written was correct and should be left alone.

Reply:
The way it was written before did not work. If a `ntemplate` file was dependent on another `ntemplate` file the cmake system would just error.

Reply:
The solution to this issue was added by @kgugala in #594, you don't need invent another one.